jellyfin-core/src/main/kotlin/org/jellyfin/sdk/discovery/DiscoveryService.kt

```diff
@@ -22,44 +20,34 @@ public class DiscoveryService(
 	}
 
 	/**
-	 * Parses the given [input] and tries to fix common mistakes.
+	 * Parses the given [input] and tries to fix common mistakes. The returned candidates are in order
+	 * of most secure followed by most likely to work.
 	 *
-	 *   - Add protocol (https and http)
-	 *   - Add port (8096 for http and 8920 for https)
-	 *   - Add baseurl (/jellyfin)
-	 *
-	 * The returned candidates are in order of most secure followed by most likely to work.
 	 * See [AddressCandidateHelper] for more information.
 	 */
 	public fun getAddressCandidates(input: String): List<String> = AddressCandidateHelper(input).run {
 		addCommonCandidates()
-		prioritize()
 		getCandidates()
 	}
 
 	/**
-	 * Connects to the [servers] and returns a list of responding servers ordered by input order.
-	 * Uses multiple rules to determine the recommendation like:
-	 *
-	 *   - HTTPS above HTTP
-	 *   - Higher Jellyfin version
-	 *   - Respond time
-	 *
-	 * Also adds the addresses reported by the server into the mix when visiting the public server
-	 * information. Reported addresses are inserted before their parent server.
+	 * Connects to the [servers] and returns a flow of responding servers according to the input order.
+	 * Uses the [RecommendedServerDiscovery] rules to determine the recommendation.
```

Comment:
Actually, it's not necessarily responding ones.

```suggestion
	 * Connects to the [servers], rates them and returns the results as a flow, in the same order as the input.
	 * Uses the [RecommendedServerDiscovery] rules to determine the recommendation.
```

Reply:
I like your change even more :+1: